src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/DoctrineTransport.php

```diff
@@ -80,9 +80,11 @@ public function setup(): void
     /**
      * Adds the Table to the Schema if this transport uses this connection.
      */
-    public function configureSchema(Schema $schema, DbalConnection $forConnection): void
+    public function configureSchema(Schema $schema, DbalConnection $forConnection/* , \Closure $isSameDatabase */): void
     {
-        $this->connection->configureSchema($schema, $forConnection);
+        $isSameDatabase = 2 < \func_num_args() ? func_get_arg(2) : static fn () => false;
```

Comment:
@nicolas-grekas do we need a runtime deprecation?

Reply:
I added an `@param` annotation to signal this